api/src/test/resources/open_lineage/event_dataset_event.json

```diff
@@ -0,0 +1,22 @@
+{
+  "eventTime": "2020-12-28T19:52:00.001+10:00",
```

Comment:
Should the dataset event have an event type?

Reply:
This is tricky, as the current meaning of `eventType` contains rather job statuses like `start`, `success` or `failed`. So it does not fit into this context and we decided not to use it to distinguish between `RunEvent`, `DatasetEvent` and `JobEvent`.